pallets/fiat_filter/src/lib.rs

```diff
@@ -241,3 +233,17 @@ impl<T: Config> Module<T> {
         };
     }
 }
+
+trait DivCeil {
```

Comment:
Can you move this to `common` pallet so other pallets can use as well? You can split that pallet in `traits` and `arith-utils`.